src/rpc/rawtransaction.cpp

```diff
@@ -493,6 +518,7 @@ UniValue getrawtransaction(const UniValue& params, bool fHelp)
             "  \"confirmations\" : n,            (numeric) The confirmations\n"
             "  \"time\" : ttt,                   (numeric) The transaction time in seconds since epoch (Jan 1 1970 GMT)\n"
             "  \"blocktime\" : ttt               (numeric) The block time in seconds since epoch (Jan 1 1970 GMT)\n"
+            "  \"feePaid\" : ttt                 (numeric, optional) The fee paid by the transaction, in satoshis.\n"
```

Comment:
Document when this field will be omitted.

Reply:
“When we can’t figure out how to populate it” 😆